libs/wingc/src/type_check/lifts.rs

```diff
@@ -0,0 +1,186 @@
+use std::collections::{BTreeMap, BTreeSet};
+
+use itertools::Itertools;
+
+use crate::ast::Symbol;
+
+/// Jsification context at the class level
+#[derive(Debug)]
+pub struct Lifts {
```

Comment:
This thing contains a lot of maps/trees and the line comments aren't enough for me to understand why all this stuff is needed. Also why do we need the `diabled` feature here. I have a feeling this can benefit from some verbose docing.

Reply:
Removed `disabled` and added some docs